tests/action-raw-offset/test-action-raw-offset.yaml

```diff
@@ -0,0 +1,44 @@
+{{- $sector1 := 80 -}}
+{{- $offset1 := 40960 -}}
+{{- $sector2 := 82 -}}
+
+{{- $image_test := "test.img" -}}
+{{- $sample := "overlays/test/test-blk.raw" -}}
+
+architecture: arm64
+
+actions:
+  - action: overlay
+    description: Deploy test block
+    source: overlays/test
+
+  - action: image-partition
+    description: Create {{ $image_test }}
+    imagename: {{ $image_test }}
+    imagesize: {{ sector 128 }}
+    partitiontype: msdos
+
+# By Byte
```

Comment:
to clean these up, move the `description` to after `action` and remove the comments as they're just a duplicate of the description

Reply:
done